website/content/guides/1.7-plugin-upgrade/index.mdx

```diff
@@ -138,11 +140,22 @@ func main() {
 
 ```
 
-So the implementation is extremely similar, but now we use the "NewSet" function, and call Run() instead of Serve().
+The implementation is extremely similar, but now we use the "NewSet" function to create the plugin Server, and call Run() instead of Serve().
+
+With the multi-plugin binary example, it is important to not that the name of the binary must be in the form of `packer-pluign-bar`. Otherwise Packer will not register it as a multiple plugin.
+
+```go
+go build -o packer-plugin-bar
+```
+
+Then you'd install it by putting it into the plugin directory with the name `packer-plugin-bar`.
+
+With the multi-plugin binary you would then refer to your newly installed plugin as `bar-foo` within your Packer template. Whereas, in the single binary setup, you'd have used the name "bar" in your Packer template.
 
-With this example, you'd install it by putting it into the plugin directory with the name `packer-plugin-bar`.
+At this point it is important to note that multi-plugins are to Packer, what providers are to Terraform. Multi-plugin binaries are meant to provide one or more plugin types to Packer. See [registering multiple plugins](#registering-multiple-plugins) for details on how to register more than one plugin type.
```

Comment:
```suggestion
At this point it is important to note that multi-plugins are to Packer, what providers are to Terraform. Multi-plugin binaries are meant to provide one or more plugin types to Packer which depend on the same cloud or other APIs. See [registering multiple plugins](#registering-multiple-plugins) for details on how to register more than one plugin type.
```

Reply:
If we add this we should we also probably add a note that it's not possible to configure plugins.( read this [proposal](https://github.com/hashicorp/terraform-proposals/blob/main/module-provider-config/module-providers.md) for more info on what I meant. )

Reply:
Good call out @azr let me see how to word this. I also wonder if this is text that we should put on the installing plugins docs as the ability to configure plugins in more in tune with the required_plugins block. At least that's my understanding.